model/polymodel.cpp

```diff
@@ -1816,23 +1807,23 @@ int ReadNewModelFile(int polynum, CFILE *infile) {
         pm->submodel[i].keyframe_matrix = (matrix *)mem_malloc((pm->submodel[i].num_key_angles + 1) * sizeof(matrix));
         if (timed) {
           pm->submodel[i].rot_start_time = (int *)mem_malloc((pm->submodel[i].num_key_angles + 1) * sizeof(int));
-          ASSERT(pm->submodel[i].rot_start_time != NULL);
+          ASSERT(pm->submodel[i].rot_start_time != nullptr);
 
           int num_ticks = (pm->submodel[i].rot_track_max - pm->submodel[i].rot_track_min);
 
           if (num_ticks > 0) {
             pm->submodel[i].tick_ang_remap = (uint16_t *)mem_malloc(num_ticks * 2);
             ASSERT(pm->submodel[i].tick_ang_remap);
           } else {
-            pm->submodel[i].tick_ang_remap = NULL;
+            pm->submodel[i].tick_ang_remap = nullptr;
           }
         }
 
-        ASSERT(pm->submodel[i].keyframe_axis != NULL);
-        ASSERT(pm->submodel[i].keyframe_angles != NULL);
-        ASSERT(pm->submodel[i].keyframe_matrix != NULL);
+        ASSERT(pm->submodel[i].keyframe_axis != nullptr);
+        ASSERT(pm->submodel[i].keyframe_angles != nullptr);
+        ASSERT(pm->submodel[i].keyframe_matrix != nullptr);
 
-        for (int t = 0; t < pm->submodel[i].num_key_angles; t++) {
+        for (t = 0; t < pm->submodel[i].num_key_angles; t++) {
```

Comment:
I'm really not fond of loop variables declared outside of said loop

Reply:
Same as earlear - out of loop usage in function.